tool/tsh/common/vnet.go

```diff
@@ -0,0 +1,94 @@
+// Teleport
+// Copyright (C) 2024 Gravitational, Inc.
```

Comment:
Friendly reminder to update your templates (if any) to 2025. I suppose this was written in 2024 so it's probably fine? (Obligatory I'm not a lawyer.)